.github/workflows/sast.yml

```
@@ -36,10 +36,10 @@
       with:
         languages: ${{ matrix.language }}
 
-    - name: Setup JDK 8
+    - name: Setup JDK 17
       uses: actions/setup-java@v2
```

actionlint

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)